src/pages/fileCentricCart/cartView.js

```diff
@@ -326,28 +360,80 @@ const cartView = ({ classes, data, isLoading }) => {
             </div>
           </div>
 
+          <div className={classes.topButtonGroup} style={divStyle(isLoading)}>
+            <button
+              type="button"
+              style={btnStyle}
+              ref={downloadButtonTop}
+              onClick={() => downloadJson(globalData, comments)}
+            >
+              {myFilesPageData.downButtonText}
+              {' '}
+            </button>
+            {' '}
+            <button
+              type="button"
+              style={btnStyle}
+              ref={deleteButtonTop}
+              onClick={removeSubjects}
+            >
+              {myFilesPageData.deleteButtonText}
+            </button>
+            <IconButton aria-label="help">
+              <HelpIcon className={classes.helpIcon} onMouseEnter={() => toggleMessageStatus('top', 'open')} onMouseLeave={() => toggleMessageStatus('top', 'close')} />
+            </IconButton>
+            { TopMessageStatus ? (
+              <div className={classes.messageTop}>
+                {' '}
+                <Message data={messageData} />
+                {' '}
+              </div>
+            ) : ''}
+          </div>
           <div id="table_selected_files" className={classes.tableWrapper}>
             {dataTable}
-            <div style={divStyle(isLoading)}>
-              <button
-                type="button"
-                style={btnStyle}
-                ref={downloadButton}
-                onClick={() => downloadJson(globalData, comments)}
-              >
-                {myFilesPageData.downButtonText}
-                {' '}
-                download manifest
-              </button>
-              {' '}
-              <button
-                type="button"
-                style={btnStyle}
-                ref={deleteButton}
-                onClick={removeSubjects}
-              >
-                {myFilesPageData.deleteButtonText}
-              </button>
+            <div className={classes.bottomButtonGroup} style={divStyle(isLoading)}>
+              <div>
+
+                <div className={classes.manifestButtonGroup}>
+                  <button
+                    type="button"
+                    style={btnStyle}
+                    ref={downloadButtonBottom}
+                    onClick={() => downloadJson(globalData, comments)}
+                  >
+                    {myFilesPageData.downButtonText}
+                    {' '}
+                  </button>
+                  {' '}
+                  <button
+                    type="button"
+                    style={btnStyle}
+                    ref={deleteButtonBottom}
+                    onClick={removeSubjects}
+                  >
+                    {myFilesPageData.deleteButtonText}
+                  </button>
+                  <IconButton aria-label="help">
```

Comment:
Is there a thought on creating this as a separate component as mouse over onMouseEnter , onMouseLeave this can be used on cases page?